doc/rtd/explanation/first_boot.rst

```diff
@@ -0,0 +1,91 @@
+.. _First_boot_determination:
```

Comment:
Did you change any of the text on this page? Just wondering if I should be paying attention to anything specific.

Reply:
Nope!